### PR TITLE
New Icon: Payhip

### LINKED ISSUE #8244: Icon Request: Payhip
### Brand Name

Payhip

### Website

https://payhip.com/

### Popularity Metric

The SimilarWeb rank is, 21332

### Official Resources for Icon and Color

color: #558dde, from main logo

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8244 

**Similarweb rank:** 21,332 (https://www.similarweb.com/website/payhip.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Hex value taken from first color in icon gradient.
Pulled Logo from SVG banner. 
Added 0.5 pt gap between heart in the process of monochrome
payhip

### CODE REVIEW

**Comment by ripzerotwo** [comment]
Note requester specified a different color however on inspection of the svg file taken from the website the left gradient color was different than reported.

File: icons/payhip.svg
Comment (adamrusted) [inline icons/payhip.svg:1]: I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/d5b6e1a0-2c1a-4e44-aa23-41c56078f8ea) when comparing your PR (black) to the icon available in their header (with a 1px gap between the shapes) in red. Would you mind taking another look @ripzerotwo?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8336,6 +8336,11 @@
             "source": "https://www.patreon.com/brand/downloads",
             "guidelines": "https://www.patreon.com/brand/downloads"
         },
+        {
+            "title": "Payhip",
+            "hex": "3197D6",
+            "source": "https://payhip.com/"
```
Comment (adamrusted) [inline _data/simple-icons.json:8342]:
```suggestion
            "source": "https://payhip.com"
```
Removal of trailing `/`

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/payhip.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Payhip</title><path d="M7 10.2h1.8c.3 0 .5-.3.5-.5V7.9c0-.3-.3-.5-.5-.5H7c-.3 0-.5.3-.5.5v1.8c0 .3.3.5.5.5zM3.7 0C1.7 0 0 1.7 0 3.6v13C0 20.7 3.3 24 7.3 24h13c2 0 3.7-1.6 3.7-3.7V0H3.7zm2.1 7c1.4-1.6 3.9-1.7 5.5-.3l.7.7-5.5 5.5-.7-.7c-1.4-1.4-1.4-3.8 0-5.2zM12 18.4l-5.5-5.5L12 7.4l5.5 5.5-5.5 5.5zm6.7-6.7-.6.6-5.5-5.4.6-.7c1.5-1.5 4-1.5 5.5 0s1.5 4 0 5.5z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/payhip.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Payhip</title><path d="M3.695 0A3.696 3.696 0 0 0 0 3.695v12.92A7.384 7.384 0 0 0 7.385 24h12.92A3.696 3.696 0 0 0 24 20.305V0H3.695zm11.653 5.604a3.88 3.88 0 0 1 .166 0 3.88 3.88 0 0 1 2.677 1.132 3.88 3.88 0 0 1 0 5.48l-.36.356c-1.826-1.825-3.648-3.656-5.476-5.482l.358-.354a3.88 3.88 0 0 1 2.635-1.132zm-6.627.125a3.88 3.88 0 0 1 2.566 1c2.068 2.062 4.127 4.133 6.192 6.199l-5.481 5.482-6.19-6.203C3.549 9.7 5.346 5.702 8.722 5.729zm-1.744 1.71a.464.464 0 0 0-.465.465v1.817c0 .256.208.463.465.463h1.816a.464.464 0 0 0 .463-.463l.008-1.817A.464.464 0 0 0 8.8 7.44H6.977z"/></svg>
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -8336,6 +8336,11 @@
             "source": "https://www.patreon.com/brand/downloads",
             "guidelines": "https://www.patreon.com/brand/downloads"
         },
+        {
+            "title": "Payhip",
+            "hex": "3197D6",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:8341]: From the website's stylesheet, I think maybe the colour should be `#5c6ac4`
```suggestion
            "hex": "5C6AC4",
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
I fixed the misalignment of the `rect` with the other elements of the heart so there may be some minor differences to the original because of that.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**